news/8808.bugfix

```diff
@@ -0,0 +1,47 @@
+Fix `tmpdir` fixture when user contains backslash
+
+On some *nix systems bound to AD domains, the $USER environment is of
+the form `DOMAIN\username`, which generates a tmpdir of the form:
+
+/tmp/pytest-of-DOMAN\username/pytest-XX/test_nameX
```

Comment:
This should be a single line entry -- the contents of this file directly go into https://pip.pypa.io/en/latest/news/.